src/js/components/PodSpecView.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+
+import DescriptionList from './DescriptionList';
+import PodSpec from '../structs/PodSpec';
+import Icon from './Icon';
+
+const METHODS_TO_BIND = [
+];
+
+class PodSpecView extends React.Component {
```

Comment:
Could you please rename this into `PodConfigurationView` or create an issue to make sure we adjust the "service" configuration view naming. Let's make sure we keep the component names in sync.